docs/en/reference/events.rst

```diff
@@ -860,6 +860,41 @@ you need to map the listener method using the event type mapping:
             /** @PostLoad */
             public function postLoadHandler(User $user, LifecycleEventArgs $event) { // ... }
         }
+
+    .. code-block:: attribute
+
+        <?php
+        use Doctrine\ORM\Event\PreUpdateEventArgs;
+        use Doctrine\ORM\Event\PreFlushEventArgs;
+        use Doctrine\Persistence\Event\LifecycleEventArgs;
+
+        class UserListener
+        {
+            #[PrePersist]
+            public function prePersistHandler(User $user, LifecycleEventArgs $event) { // ... }
```

Comment:
Shall we make the return type explicit to indicate that we don't expect the listener to return anything?

```suggestion
            public function prePersistHandler(User $user, LifecycleEventArgs $event): void { // ... }
```